core/src/main/java/com/linecorp/armeria/server/http/dynamic/DynamicHttpServiceBuilder.java

```diff
@@ -0,0 +1,232 @@
+/*
+ * Copyright 2016 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.server.http.dynamic;
+
+import static java.util.Objects.requireNonNull;
+
+import java.lang.reflect.Method;
+import java.util.ArrayList;
+import java.util.EnumSet;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+import java.util.concurrent.Executor;
+import java.util.concurrent.ForkJoinPool;
+
+import com.google.common.collect.Sets;
+
+import com.linecorp.armeria.common.http.HttpMethod;
+
+/**
+ * Builds a new {@link DynamicHttpService}.
+ */
+public final class DynamicHttpServiceBuilder {
+
+    private Executor executor = ForkJoinPool.commonPool();
```

Comment:
We already have `ServiceRequestContext.blockingTaskExecutor()`. No need to have an `executor` property.